route4me-csharp-sdk/Route4MeSDKLibrary/Route4MeManagerV5.cs

```diff
@@ -955,12 +964,16 @@ public ResultResponse BulkCreateTeamMembers(TeamRequest[] membersParams, out Res
         /// <returns>List of the driver reviews</returns>
         public Task<Tuple<DriverReviewsResponse, ResultResponse>> GetDriverReviewListAsync(DriverReviewParameters parameters)
         {
-            return GetJsonObjectFromAPIAsync<DriverReviewsResponse>(parameters,
+            var result = GetJsonObjectFromAPIAsync<DriverReviewsResponse>(parameters,
                 R4MEInfrastructureSettingsV5.DriverReview,
                 HttpMethodType.Get,
                 null,
                 true,
                 false);
+
+            return Task.FromResult(
```

Comment:
What is the purpose of this change? I would recommend to avoid unnecessary memory traffic here. You can just return result or revert changes.